3dt.go

```diff
@@ -70,6 +71,17 @@ func main() {
 	log.Info("Start 3DT")
 	go api.StartUpdateHealthReport(dt, readyChan, false)
 	router := api.NewRouter(dt)
-	log.Infof("Exposing 3DT API on 0.0.0.0:%d", config.FlagPort)
-	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", config.FlagPort), router))
+
+	// try using systemd socket
+	listeners, err := activation.Listeners(true)
+	if err != nil {
+		log.Error(err)
+	}
+
+	if len(listeners) == 0 || listeners[0] == nil {
+		log.Error("Could not find listen socket")
+		log.Infof("Exposing 3DT API on 0.0.0.0:%d", config.FlagPort)
```

Comment:
Do we want to also inform the user here that it's a Unix socket to ensure there isn't any confusion?